### ISSUE TITLE
A11y_mssql for VSCode_Message Window_ColorContrast:The color contrast ratio of shortcut "Crtl+Alt+Y" to the background is 3.164:1

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17195]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev2;#ColorContrast;#AINotInScope;#WCAG1.4.3;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11

## Repro Steps:

1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Press enter “New Query” write any valid query and click on "Execute query button"
7. Message Window will open.
8. Open Color contrast analyzer and check if color contrast ratio of shortcut "Crtl+Alt+Y" to the background is in ratio of 4.5:1.

## Actual:
The color contrast ratio of shortcut "Crtl+Alt+Y" to the background is 3.164:1

**Similar issue is also observed in Select top 1000 Results window_shortcut "Crtl+Alt+R" to the background.**

## Expected:
The color contrast ratio of shortcut "Crtl+Alt+Y" to the background should be 4.5:1

## User Impact:
Low vision users will find it difficult to view the shortcut "Crtl+Alt+Y"
## Attachments
![17195_A11y_mssql for VSCode_Message Window_ColorContrast](https://user-images.githubusercontent.com/86601127/149115519-2598ec0b-b6e0-4a52-8bea-6c04da695866.png)

**Comment by Roopesh-T** [comment]
![image](https://user-images.githubusercontent.com/86601127/178235261-506aeddc-2c37-4443-95c6-629913e55480.png)
issue does not repro in the env so closing the bug.
